vec_processing/topics_nearest_doc.py

```diff
@@ -67,4 +67,18 @@ def split_2d_list(word_vecs):
     return vecs, ids
 
 def insert_nearest_docs(api_url, storage_path):
-    print('not implemented')
+    nearest_docs_data = load_json_data(storage_path+NEAREAST_DOCS_FILE_NAME)
+    delete_nearest_docs(api_url)
+    nearest_docs = []
+    for arts in nearest_docs_data:
```

Comment:
De to loops kan skrives meget kortere. Jeg syndes i skal flytte de to loops ind i function `insert_nearest_arts` og sende  `nearest_docs_data` med som parametre